### PR TITLE
Bug 1925411: bump(*): k8s 1.21.0-beta.0

### PR DESCRIPTION
**Pull request by gabemontero**
/assign @adambkaplan 

it vendored in and compiled / passed unit tests with no problem

as we discussed in grooming, we could do this now, and then still do or normal bump to 1.21 (at a later level) via a Jira as part of our normal per release iteration on k8s levels

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -236,6 +234,8 @@ github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d h1:3PaI8p3seN09Vjb
 github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d/go.mod h1:SlYgWuQ5SjCEi6WLHjHCa1yvBfUnHcTbrrZtXPKa29o=
 github.com/gogo/protobuf v1.3.1 h1:DqDEcV5aeaTmdFBePNpYsp3FlcVH/2ISVVM9Qf8PSls=
 github.com/gogo/protobuf v1.3.1/go.mod h1:SlYgWuQ5SjCEi6WLHjHCa1yvBfUnHcTbrrZtXPKa29o=
+github.com/gogo/protobuf v1.3.2 h1:Ov1cvc58UF3b5XjBnZv7+opcTcQFZebYjWzi34vdm4Q=
```
Comment (gabemontero) [inline go.sum:237]: this is the change proscribe by the upstream k8s/k8s issue noted in the BZ

**Comment by gabemontero** [comment]
okd installs failing.  Moving on

/skip

**Comment by gabemontero** [comment]
samples was fine in failing upgrade job

/test e2e-aws-upgrade

**Comment by gabemontero** [comment]
other operators blocking upgrade job

/test e2e-aws-upgrade

**Comment by adambkaplan** [comment]
/hold

I'm a bit leery of having k8s 1.21 clients running against k8s 1.20 apiservers. Also noted in the BZ, we need updates to generated protobuf files from openshift/api in addition to updated k8s protobuf files.

Unfortunately it seems that we may need to sit on this until the rebase lands.

**Comment by gabemontero** [comment]
> /hold
> 
> I'm a bit leery of having k8s 1.21 clients running against k8s 1.20 apiservers. Also noted in the BZ, we need updates to generated protobuf files from openshift/api in addition to updated k8s protobuf files.

I missed the updated protobuf bits in the BZ.

Yeah I'm going to close this PR then and wait until openshift/api is bumped.


> 
> Unfortunately it seems that we may need to sit on this until the rebase lands.

**Pull request closed without merging**